tensorflow/core/common_runtime/threadpool_device.h

```diff
@@ -44,6 +44,8 @@ class ThreadPoolDevice : public LocalDevice {
 
   Status Sync() override { return Status::OK(); }
 
+  bool IsRemoteCallAllowed() const { return true; }
```

Comment:
Is this override needed?

Reply:
Right, seems to be of CPU type so it's handled by whitelist.